### PR TITLE
Igorjosesantos/c01 git04

### PR DESCRIPTION
**Pull request by igorjosesantos**
# Description

Exercise (e.g. C01-E01): 

Student Name:

> A brief explanation of what has been done.

## Type of change

- [x] Exercise Submission (one exercise per PR)
  - [x] This PR is just for **one** exercise of a class
  - [x] PR name follows the pattern `<github-username>/<exercise-number>`
  - [x] Added exercise files inside folder `/classes/<class- number>/exercises/<exercise-number>/<github-username>/`
	- [x] Review [exercise submission steps](./README.md#Exercises)
- [ ] Class content (instructors/admins)
- [ ] Documentation update
- [ ] Repo management

### CODE REVIEW

File: classes/01class/exercises/c01-git04/igorjosesantos/README.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+5. Which commands would you use? Explain what the command does.
+
+```
+# Using `filter-branch`
+
+To completely remove the entire file from the history
+
+```
+$ git filter-branch --force --index-filter \
```
Comment (denstorti) [inline classes/01class/exercises/c01-git04/igorjosesantos/README.md:29]: `git filter-repo` is the new tool recommended for the job by git-scm. Simpler to use and safer 👍

Comment (igorjosesantos) [reply classes/01class/exercises/c01-git04/igorjosesantos/README.md:29]: I see, however, do you agree it doesn't mean that won't work?!

Comment (igorjosesantos) [reply classes/01class/exercises/c01-git04/igorjosesantos/README.md:29]: I see. You're right. (https://github.com/newren/git-filter-repo)

However, do you agree it doesn't mean that won't work?!

Comment (igorjosesantos) [reply classes/01class/exercises/c01-git04/igorjosesantos/README.md:29]:
```
git filter-repo --path "my_env.txt" --invert-paths
git push --force
```

Comment (denstorti) [reply classes/01class/exercises/c01-git04/igorjosesantos/README.md:29]: filter-branch is still a valid answer.

**Comment by denstorti** [comment]
@igorjosesantos , is this exercise about git03 or git04?
There is no description and the answer is not following the pattern.

Please fix the branch for answering just one exercise at a time.

**Comment by igorjosesantos** [comment]
.

**Pull request closed without merging**